README.md

```diff
@@ -281,6 +390,54 @@ $ ./cjdroute --bench | grep "per second"
 1488764674 INFO Benchmark.c:62 Benchmark Switching in 3987ms. 51366 packets per second
 ```
 
+## ODROID C2
+
+Both builds were performed on Armbian, Armbian_5.25_Odroidc2_Debian_jessie_default_3.14.79.img using cjdns-v19.1
+
+``` 
+$ uname -a
+Linux odroidc2 3.14.79-odroidc2 #32 SMP PREEMPT Wed Feb 1 22:28:54 CET 2017 aarch64 GNU/Linux
+
+$ gcc --version | grep "gcc" 
+gcc (Debian/Linaro 4.9.2-10) 4.9.2
+
+$ ./cjdroute --version 
+Cjdns version: cjdns-v19.1
+Cjdns protocol version: 19
+```
+
+### Default Build
+
+#### Bench
+
+```
+$ MINVER=0.0.0 ./do
+
+$ ./cjdroute --bench | grep "per second"
+1489380110 INFO Benchmark.c:62 Benchmark salsa20/poly1305 in 4651ms. 258009 kilobits per second
```

Comment:
@cjdelisle @Famicoman salsa20/poly1305 is 5x and switching is 3x faster on my Macbook Pro. These boards are catching up.